http-client/src/main/java/io/avaje/http/client/DUrlBuilder.java

```diff
@@ -15,13 +15,21 @@ final class DUrlBuilder implements UrlBuilder {
 
   @Override
   public UrlBuilder url(String url) {
-    buffer.delete(0, buffer.length());
+
+    if (url.startsWith("http") && url.contains("://")) {
+      buffer.setLength(0);
+    }
+
     buffer.append(url);
     return this;
   }
 
   @Override
   public UrlBuilder path(String path) {
+    if (path.startsWith("http") && path.contains("://")) {
+      buffer.setLength(0);
+      return this;
```

Comment:
this does not look right?  the path is not appended before the `return this` ?

Reply:
oh woops